op-guide/requirement.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Software and Hardware Requirements
+category: deployment
+---
+
+# TiDB Software and Hardware Requirements
+
+## About
+
+As an open source distributed NewSQL database with high performance, TiDB can be deployed in the Intel architecture server and mainstream virtualization environments and runs well. TiDB supports most mainstream hardware networks and the mainstream Linux operating system. 
+
+## Linux OS Version Requirements
+
+| Linux OS Platform        | Version      |
+| :-----------------------:| :----------: |
+| Red Hat Enterprise Linux | 7.3 and above|
+| CentOS                   | 7.3 and above|
+| Oracle Enterprise Linux  | 7.3 and above|
+| Ubuntu LTS               | 14.04 and above|
+
+**Note**
+
+- The Oracle Enterprise Linux only supports the Red Hat Compatible Kernel (RHCK), and does not support the Unbreakable Enterprise Kernel provided by Oracle Enterprise Linux.
```

Comment:
The Oracle Enterprise Linux only supports the Red Hat Compatible Kernel (RHCK), and does not support the Unbreakable Enterprise Kernel provided by Oracle Enterprise Linux. 
Please double check... if it needs to be changed to
-> For Oracle Enterprise Linux, TiDB supports the Red Hat Compatible Kernel (RHCK) and does not support the Unbreakable Enterprise Kernel provided by Oracle Enterprise Linux.